### PR TITLE
Change log level to error for merge failures

### PR DESCRIPTION
**Pull request by nathan-artie**

### CODE REVIEW

File: processes/consumer/flush.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func Flush(ctx context.Context, inMemDB *models.DatabaseData, dest destination.B
 
 			if err != nil {
 				tags["what"] = "merge_fail"
-				slog.With(logFields...).Warn(fmt.Sprintf("Failed to execute %s...not going to flush memory", action), slog.Any("err", err))
+				slog.Error(fmt.Sprintf("Failed to execute %s...not going to flush memory", action), append(logFields, slog.Any("err", err))...)
```
Comment (Tang8330) [inline processes/consumer/flush.go:93]: Does append work?

Comment (Tang8330) [reply processes/consumer/flush.go:93]: Also, any reason not to just change `.Warn` to `.Error`? 

```go
slog.With(logFields...).Error(fmt.Sprintf("Failed to execute %s...not going to flush memory", action), slog.Any("err", err))
```

Comment (nathan-artie) [reply processes/consumer/flush.go:93]: It should https://go.dev/play/p/yeI7nHhZ9X8?v=goprev

Comment (nathan-artie) [reply processes/consumer/flush.go:93]: I can just do the simple level change for now.

**Review by Tang8330 — APPROVED** [review]

File: processes/consumer/flush.go (hunk 1 above)
Comment (Tang8330) [reply processes/consumer/flush.go:93]: Good to know

**Pull request merged**